web/src/css/6-components/_know-your-rights-route.scss

```diff
@@ -0,0 +1,21 @@
+.RightsList {
+    font-family: Times;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    padding: 0.6rem;
+}
+
+.RightsItem h6 {
```

Comment:
In general, tag based selectors like`h6` aren't used anymore. I think it's because they easily bleed over onto unintended elements. I would probably change this whole selector to be `.RightsItem-header` and apply that class appropriately in the markup.